api/routes.py

```diff
@@ -0,0 +1,66 @@
+from flask import jsonify
+from flask.views import MethodView
+from flask_smorest import Blueprint
+from sqlalchemy import desc, text
+from api import db
+from api.schemas import *
```

Comment:
It's better to avoid wildcard imports for the sake of code readability/clarity, even when effectively importing _everything_ from another file. Either do `from api.schemas import X, Y, Z,...` or list one import per line:
```
from api.schemas import X
from api.schemas import Y
from api.schemas import Z
...
```

Reply:
I'll sort that.  Regarding deleting the empty folders:  I did!  They aren't present in this branch.  Or, well, I don't see them anymore, anyway.

Reply:
> I'll sort that. Regarding deleting the empty folders: I did! They aren't present in this branch. Or, well, I don't see them anymore, anyway.

You're right! They're ghosts in my dev environment 👻